client/components/App.jsx

```diff
@@ -22,7 +22,7 @@ class Host extends React.Component {
   }
 
   componentDidMount() {
-    const {listingId} = this.state;
+    const listingId = window.location.pathname.slice(1, -1);
     // const url = `http://${window.location.host}/api/location/${listingId}`;
```

Comment:
why the switch to listingIds?

Reply:
I need to get the total number of reviews per host to render that number. My database has an array of listings that the host owns.